cl/api/templates/rest-docs-vlatest.html

```diff
@@ -24,7 +24,7 @@ <h4 class="v-offset-below-2">
     {% include "includes/toc_sidebar.html" %}
   </div>
   <div class="col-xs-12 col-md-8 col-lg-6">
-    <h1 id="about">REST API &ndash; v4.0</h1>
+    <h1 id="about">REST API &ndash; v4.1</h1>
```

Comment:
Let's catch this in the HTML title too.

Reply:
Thanks for bringing this to my attention.